faiss/python/swigfaiss.swig

```diff
@@ -288,12 +288,13 @@ namespace std {
 %include <faiss/utils/hamming.h>
 %include <faiss/utils/hamming_distance/common.h>
 
```

Comment:
This fixed an issue where you could not build with the python option on and gpu option off. I/We can back this out if it's out of scope for this effort.